openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/penalites.py

```diff
@@ -1,92 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Thanks @laem, I think an issue with a reference to the commit does the job well. A PR would have been kind of too much, as it would stay open for a long time.